config/packages.yaml

```diff
@@ -1,5 +1,4 @@
 packages:
   all:
     providers:
-      tbb: [fairsoft_backports.intel-tbb]
```

Comment:
I guess, we need a `vae/config/` directory now?
@kresan ?

Reply:
done